static/js/publisher/release/actions/pendingReleases.test.js

```diff
@@ -225,4 +227,27 @@ describe("pendingReleases actions", () => {
       expect(resumeProgressiveRelease(key).payload).toBe(key);
     });
   });
+
+  describe("cancelProgressiverelease", () => {
+    const key = "progressive-test";
+    const previousRevision = "previous-revision";
```

Comment:
Would be good for this to be the same as in real case.
Is `previousRevision` supposed to be revision id? Whole revision object? This test doesn't tell which one.